src/components/TopicOfArticles.jsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from "react";
+import {getArticles, getUsers} from '../utils/api'
+import {useParams} from 'react-router-dom'
+
+const TopicOfArticles = () => {
+   
+const {topic} = useParams()
+
+const [articles, setArticles] = useState([])
+const [users, setUsers] = useState([])
+useEffect(() => {
+    getArticles(topic).then((article) => {
+        
+        setArticles(article.articles)
+    })
+},[])
```

Comment:
when would we want to change what was displayed on the page? Maybe when topic is changed? Take advantage of the dependency array here